setup.py

```diff
@@ -23,10 +23,13 @@
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
 
+# TODO: update cylc/cylc-flow dependency after cylc-flow PR has been merged
 install_requires = [
     'jupyterhub==1.0.*',
     'tornado==6.0.*',
-    'graphene-tornado==2.1.*'
+    'graphene-tornado==2.1.*',
+    ('cylc @ https://github.com/dwsutherland/cylc'
+     '/tarball/protobuf-uis-feed#egg=cylc-8.0a1.dev')
```

Comment:
Sorry, there is one more thing necessary to get `setup.py` working.

If you install this branch with `pip install -e .` everything should work. But if you try `pip install .` it will fail to run. That's because editable installations are links back to the project source, which gets included in the `sys.path`.

Whereas installing via `pip install .`, it will read the package contents, and include only what's described there. In our case, because we started without a package/folder, we had to manually list what was included.

See further down in this file, and you should find a key in `setup(` similar to this one:

```
setup(
...
py_modules=['handlers', 'cylc_singleuser'],
...
)
```

We need to list each file that will be included in the package, manually, one by one there. Painful I know, but once the GraphQL work is merged we can now create `cylc.uiserver` structure, and re-organize files.